### PR TITLE
fix recent blog component

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/3662

### CODE REVIEW

File: network-api/networkapi/settings.py [hunk 1]
```diff
@@ -269,6 +269,7 @@
             ])),
             'libraries': {
                 'bg_nav_tags': 'networkapi.buyersguide.templatetags.bg_nav_tags',
+                'blog_tags': 'networkapi.wagtailpages.templatetags.blog_tags',
```
Comment (youriwims) [inline network-api/networkapi/settings.py]: ...will never forget to register tem

File: network-api/networkapi/wagtailpages/customblocks/recent_blog_entries.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ class RecentBlogEntries(blocks.StructBlock):
     def get_context(self, value, parent_context=None):
         context = super().get_context(value, parent_context=parent_context)
         IndexPage = apps.get_model('wagtailpages.IndexPage')
-        blogpage = IndexPage.objects.get(title="blog")
+        blogpage = IndexPage.objects.get(title__iexact="blog")
```
Comment (youriwims) [inline network-api/networkapi/wagtailpages/customblocks/recent_blog_entries.py]: ....you learn something new every day.

File: network-api/networkapi/settings.py (hunk 1 above)
Comment (Pomax) [reply network-api/networkapi/settings.py]: we may have to revisit whether this deserves its own file or whether it makes sense to fold it in somewhere else... (it's technically zero code related to blogs)

**Comment by Pomax** [comment]
filed https://github.com/mozilla/foundation.mozilla.org/issues/3664 as a follow-up

**Pull request merged**